main.go

```diff
@@ -257,5 +257,6 @@ func main() {
 		}
 	default:
 		fmt.Fprintf(os.Stderr, "Unknown git command %s.\n", subcommand)
+                os.Exit(1)
```

Comment:
This is a different code path than the other one that you had fixed, which was the case where you run dgit with no subcommand at all and it produces the usage. This case involves calling an unrecognized subcommand.